config.md

```diff
@@ -31,7 +31,7 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 * **`path`** (string, OPTIONAL) Specifies the path to the root filesystem for the container. The path is either an absolute path or a relative path to the bundle.
     Users SHOULD consider using a conventional name, such as `rootfs`.
 
-    On Windows, for Windows Server Containers, this field is REQUIRED. For Hyper-V Containers, this field MUST be omitted.
+    On Windows, for Windows Server Containers, this field is REQUIRED and MUST be specified as a volume GUID path. For more information, see [MSDN documentation][naming-a-volume]. For Hyper-V Containers, this field MUST be omitted.
```

Comment:
Maybe “see MSDN documentation” → “see the MSDN documentation”?  Or use:

> … MUST be specified as a [volume GUID path][naming-a-volume].

and drop the “For more information” sentence?

[naming-a-volume]: https://aka.ms/nb3hqb